server/controllers/medical/patients/index.js

```diff
@@ -475,7 +475,9 @@ function find(options) {
     parameters.push(limit);
   }
 
-  return db.exec(sql, parameters);
+  let promise = options.ID ? db.one(sql, parameters) : db.exec(sql, parameters);
```

Comment:
It would be ideal to implement this feature without changing the server side search method - this API route is used by many many modules and making it specific for this feature/ query will make it very hard to maintain in the future.